### PR TITLE
Fixed ASHRAE9012016 ApartmentMidSize and OutpatientHealthCare Test files

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request overview
---------------------
Fixed issue #7290. ASHRAE901-2016 test files ASHRAE9012016_ApartmentMidRise_Denver.idf and ASHRAE9012016_OutPatientHealthCare_Denver.idf were failing. The first one had mismatch of azimuth angle of a base and its sub-surfaces for three surfaces. The second one had two issues missing fields in ZoneEquipmentList, and incorrect SetpointManager:MixedAir.  Now these two test files run to completion. The ASHRAE9012016_Hospital_Denver.idf file runs in my machine. 

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified

### CODE REVIEW

**Comment by Nigusse** [comment]
I forgot to include the fixed test files. There were excluded from the regression tests.

**Comment by Nigusse** [comment]
I forced the "ILinux-Ubuntu-18.04-gcc-7.4-IntegrationCoverage-Debug" test re-run and I still get two test files fail (timeout). This files are not related to the changes I made. @Myoldmopar any clue why this is happening?

**Comment by Myoldmopar** [comment]
@nigusse It's because these two test files take approximately one hour to run, and CI imposes a one hour limit on the tests.  Sometimes the machine takes a smidge longer and these files jump over the one hour mark, and sometimes they run faster.  It could also depend on if the machine is busy doing other builds, running backups, etc.  No big deal, this isn't related to your work here.

File: testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf [hunk 1]
```diff
@@ -41616,6 +41616,8 @@
     Floor 1 Anesthesia VAV Box,  !- Zone Equipment 1 Name
     1,                       !- Zone Equipment 1 Cooling Sequence
     1,                       !- Zone Equipment 1 Heating or No-Load Sequence
+    ,                        !- Zone Equipment 1 Sequential Cooling Fraction
```
Comment (Myoldmopar) [inline testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf:41619]: Yeah, good call.  Must've been a bad transition.

File: testfiles/ASHRAE9012016_ApartmentMidRise_Denver.idf [hunk 2]
```diff
@@ -5586,9 +5586,9 @@
     1,                       !- Multiplier
     4,                       !- Number of Vertices
     46.327300000000,0.381000000000,2.173100000000,  !- X,Y,Z ==> Vertex 1 {m}
-    46.327300000000,0.381000000000,0.954000000000,  !- X,Y,Z ==> Vertex 2 {m}
+    46.327300000000,1.295300000000,2.173100000000,  !- X,Y,Z ==> Vertex 2 {m}
     46.327300000000,1.295300000000,0.954000000000,  !- X,Y,Z ==> Vertex 3 {m}
-    46.327300000000,1.295300000000,2.173100000000;  !- X,Y,Z ==> Vertex 4 {m}
+    46.327300000000,0.381000000000,0.954000000000;  !- X,Y,Z ==> Vertex 3 {m}
```
Comment (Myoldmopar) [inline testfiles/ASHRAE9012016_ApartmentMidRise_Denver.idf:5591]: Now that's interesting.  This worked in an older version of the file, but I guess things got more strict about the surface direction along the way.  Good find.

File: testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf [hunk 3]
```diff
@@ -48330,13 +48364,13 @@
...
-    AHU-1_HeatC-AHU-1_FanNode,  !- Fan Inlet Node Name
-    AHU-1_Fan-AHU-1 HumidifierNode,  !- Fan Outlet Node Name
-    AHU-1 Supply Equipment Outlet Node;  !- Setpoint Node or NodeList Name
+!  SetpointManager:MixedAir,
+!    AHU-1_ExtraWaterHeatC Air Temp Manager,  !- Name
+!    Temperature,             !- Control Variable
+!    AHU-1 Supply Equipment Outlet Node,  !- Reference Setpoint Node Name
+!    AHU-1_HeatC-AHU-1_FanNode,  !- Fan Inlet Node Name
+!    AHU-1_Fan-AHU-1 HumidifierNode,  !- Fan Outlet Node Name
+!    AHU-1 Supply Equipment Outlet Node;  !- Setpoint Node or NodeList Name
```
Comment (Myoldmopar) [inline testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf]: Is this kept in here for any specific reason?  If not, maybe we should just take it out.

Comment (Nigusse) [reply testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf]: No reason to keep it. Removed.

File: testfiles/ASHRAE9012016_ApartmentMidRise_Denver.idf (hunk 2 above)
Comment (Nigusse) [reply testfiles/ASHRAE9012016_ApartmentMidRise_Denver.idf:5591]: Yeah, I think these files were translated starting from version 8.6.

**Comment by Nigusse** [comment]
@Myoldmopar Addressed your review comment and merged in develop. This branch should be green if CI is happy.

**Comment by Nigusse** [comment]
@Myoldmopar This branch is ready to in.

**Comment by Myoldmopar** [comment]
Yep, good to go, thanks @Nigusse

**Pull request merged**